demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -234,8 +235,8 @@ def is_not_default_image(self):
     def is_valid_image_name(self):
         """Check if the image name is valid"""
         image_path = self.file_path
```

Comment:
can we create the Path object once, then reuse it? (instead of calling the constructor multiple times)